dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/DataQueryParams.java

```diff
@@ -636,6 +639,18 @@ public String getKey()
             .addIgnoreNull( "apiVersion", apiVersion ).build();
     }
 
+    private String getDimensionalItemKeywords( final DimensionItemKeywords keywords )
+    {
+        final StringBuilder keys = new StringBuilder();
+
+        if ( keywords != null && !keywords.isEmpty() )
+        {
+            keywords.getKeywords().forEach( k -> keys.append( "key:" ).append( "[" + k.getKey() + "]" ) );
```

Comment:
I think the `.append( "key:" )` here is not necessary, as the keys themselves are unique, and cannot match UIDs.

Reply:
Yes, I agree. It was only to keep a similar standard of the other values appended to the key. But I will remove it now. :+1: